documentation/general/workloads/cross-version-workloads.md

```diff
@@ -0,0 +1,138 @@
+    
+    end
+    
+    wasm-tools.7.0.100----->|extends|wasm-tools-net7-abstract
+    wasm-tools-net6----->|extends|wasm-tools-net6-abstract
+    
+    subgraph Mono.ToolChain.net7
+    wasm-tools-net7-abstract([wasm-tools-net7-abstract])-->Mono.browser-wasm.net7
+    Mono.browser-wasm.net7-->|alias|Mono.browser-wasm.v7["Mono.browser-wasm v7.0.0"]
+    wasm-tools-net7-abstract-->AOT.Cross.browser-wasm.net7
+    AOT.Cross.browser-wasm.net7-->|alias|AOT.Cross.browser-wasm.v7["AOT.Cross.browser-wasm v7.0.0"]
+    end
+    
+    wasm-tools-net7-abstract------>|extends|microsoft-net-sdk7-emscripten;
+    
+    subgraph workload.emscripten.net7
+    microsoft-net-sdk7-emscripten(["microsoft-net7-sdk-emscripten<BR><I>(abstract)</I>"])-->Microsoft.NET.Runtime.7.Emscripten.Node[Emscripten.Node.net7];
+    Microsoft.NET.Runtime.7.Emscripten.Node-->|alias|Microsoft.NET.Runtime.Emscripten.Node["Emscripten.Node v7.0.0"]
+    microsoft-net-sdk7-emscripten-->Microsoft.NET.Runtime.7.Emscripten.Python[Emscripten.Python.net7];
+    Microsoft.NET.Runtime.7.Emscripten.Python-->|alias|Microsoft.NET.Runtime.Emscripten.Python["Emscripten.Python v7.0.0"]
+    microsoft-net-sdk7-emscripten-->Microsoft.NET.Runtime.7.Emscripten.Sdk[Emscripten.Sdk.net7];
+    Microsoft.NET.Runtime.7.Emscripten.Sdk-->|alias|Microsoft.NET.Runtime.Emscripten.Sdk["Emscripten.Sdk v7.0.0"]
+    end
+
+```
+
+## Aliased workload packs
+
+Workload pack aliases will be used to support depending on different versions of the same workload pack ID.  This is similar to the [Side-by-Side Workload Pattern](https://github.com/dotnet/designs/blob/main/accepted/2020/workloads/workload-manifest.md#side-by-side-workload-pattern) described in the workload manifest design spec.  Using `Emscripten.Sdk` as an example, there will be a version 6.0.0 and a version 7.0.0 of the workload pack.  The `workload.emscripten.net6` manifest will alias `Emscripten.Sdk.net6` to version 6.0.0 of the package, and the `workload.emscripten.net7` manifest will alias `Emscripten.Sdk.net7` to version 7.0.0 of the package.  Then the logic which imports the workload pack can conditionally import the right version of the pack based on the target framework version.  This logic is expected to be in the `WorkloadManifest.targets` file for the workload manifest that defines the `wasm-tools` workload.  For the 7.0.100 version of that manifest, the import logic would look something like this:
+
+```xml
+  <ImportGroup Condition="'$(RunAOTCompilation)' == 'true' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(TargetFrameworkVersion)' == '6.0'">
+    <Import Project="Sdk.props" Sdk="Emscripten.Sdk.net6" />
+  </ImportGroup>
+  <ImportGroup Condition="'$(RunAOTCompilation)' == 'true' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(TargetFrameworkVersion)' == '7.0'">
+    <Import Project="Sdk.props" Sdk="Emscripten.Sdk.net7" />
+  </ImportGroup>
+```
+
+## .NET 6 manifests in .NET 7 SDK
```

Comment:
How far back does this go? Will .NET 8 support `net7.0-android` and `net6.0-android`? Or is it only have 1 LTS release behind?

Reply:
That's up to the workload.  For .NET, we still support building projects all the way back to .NET Core 1.0.  I understand that for iOS and Android this is not as important, as you need to move forward to new tooling / targeting new versions of .NET in order to continue to publish to the app stores.  So supporting one previous version for iOS and Android is probably fine.